### PR TITLE
Update to use setuptools-scm, remove current handling of version

### PR DESCRIPTION
**Pull request by jsdillon**

### CODE REVIEW

File: hera_cal/tests/test_utils.py [hunk 1]
```diff
@@ -138,6 +138,17 @@ def test_filter_bls(self):
             utils.filter_bls(bls, min_bl_cut=1.5)
 
 
+class TestHistoryVersion(object):
```
Comment (steven-murray) [inline hera_cal/tests/test_utils.py:141]: don't use object

**Review by steven-murray — COMMENTED** [review]
Looks good, one comment, plus delete `hera_cal/_version.py` and add to .gitignore. Oh, also review the README to check the installation instructions.

File: hera_cal/tests/test_utils.py (hunk 1 above)
Comment (jsdillon) [reply hera_cal/tests/test_utils.py:141]: Done.

**Review by steven-murray — APPROVED** [review]
Great!

**Pull request merged**